### DISCUSSION TITLE
When using allowlist or some filter, remove what from the title what is being filtered

### DISCUSSION THREAD

**Discussion by jnpetersen** [totaldebug/atomic-calendar-revive#1599] [Feature Requests]
In my case, my wife and I share a calendar and we mark what we plan for dinner on the calendar with it prefixed `Dinner = ` and I would like to have it remove that from the text showing so for example `Dinner = Sushi` When I go to see this specific card that is filtered for only dinners on the calendar, it will just show me `Sushi` for that day.  I already have a header that says Dinners so I only want the name of what we are having.

I had thought of having a separate calendar, but it was hard enough to get my wife to work with 1 shared calendar 😄

**Comment by jnpetersen** [comment]
And maybe having it as an option so if people want to see it, they can, but for those who don't, a toggle for "Remove filtered text" or something would be great.

**Comment by marksie1988** [comment]
Implemented on the `beta` branch in c21f64f. Combine `allowlist` (filter on the original summary) with `titleReplace` (strip the matched prefix from the display):

```yaml
entities:
  - entity: calendar.family
    allowlist: '^Dinner = '
    titleReplace:
      - from: '^Dinner = '
        to: ''
```

The card filters to events matching `^Dinner = ` and shows just the meal ("Sushi") on screen. `blocklist`/`allowlist` always run against the original summary, so the filter still fires even after the rewrite. Closing as implemented.
